snap/snapcraft.yaml

```diff
@@ -32,7 +32,8 @@ parts:
     override-build: |
       if git describe --exact-match --tags 2> /dev/null
       then
-        echo -n > ../src/prerelease.txt
+        # echo -n > ../src/prerelease.txt
+        # commented this line to stop generation of prerelease.txt. Need to check how to refactor the process to remove this line completely.
```

Comment:
Snapcraft config is not actively maintained. See https://github.com/ethereum/solidity/pull/12507#issuecomment-1011971783 for some context/discussion.

In this case the script was creating an empty `prerelease.txt` to get a release build. 
I think that the safest thing to do here is to replace this with a line that removes `prerelease.lock` - since I'm not sure if the `cmake install` command below would be the right place to add the new flag or not. At least with `prerelease.lock` we know that it will work.